lib/ghomperm.gi

```diff
@@ -1442,7 +1447,25 @@ local   D,  I,G;
     SetStabChainMutable(G,I);
     return G;
   fi;
-end );
+end;
+
+if IsBound(HPCGAP) then
+# In HPC-GAP we can't filter on HasStabChainMutable because it is thread-local
```

Comment:
Hm, but `HasStabChainMutable` *is* mutable, it just isn't a filter... Perhaps "... because StabChainMutable is a thread-local attribute, and therefore HasStabChainMutable isn't a filter (for details, see XYZ)"